<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="chartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1000</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="serialTipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>串口未开启</string>
    </property>
   </widget>
   <widget class="QLabel" name="companyLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>520</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>智澜科技</string>
    </property>
   </widget>
   <widget class="QLabel" name="taobaoLabel">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>520</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>官方淘宝商铺</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseUpdateButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>522</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>暂停更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windowTopButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>522</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>窗口置顶</string>
    </property>
   </widget>
   <widget class="QFrame" name="ch1Frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 0, 0, 70);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ch1Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ch1LcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="value" stdset="0">
      <double>1234.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1234</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ch2Frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>70</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 0, 255, 70);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ch2Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ch2LcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="value" stdset="0">
      <double>1234.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1234</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ch3Frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>130</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 128, 0, 70);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ch3Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ch3LcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="value" stdset="0">
      <double>1234.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1234</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ch4Frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>190</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 0, 255, 70);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ch4Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ch4LcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="value" stdset="0">
      <double>1234.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1234</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ch5Frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>250</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 255, 255, 70);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ch5Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ch5LcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="value" stdset="0">
      <double>1234.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1234</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="pLineEdit">
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	/* 边框1px宽，颜色为深紫色 */
    border: 1px solid rgb(41, 57, 85);
	/* 给定3px边框圆角 */
    border-radius: 3px;
	/* 背景色定为白色吧 */
    background: white;
	/* 文本的大小 */
    font-size: 14px ;
}
 
QLineEdit:hover {
	/* 鼠标悬停时，我们将编辑框的边框设置为蓝色 */
    border: 1px solid blue; 
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>比例（P）：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="iLineEdit">
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	/* 边框1px宽，颜色为深紫色 */
    border: 1px solid rgb(41, 57, 85);
	/* 给定3px边框圆角 */
    border-radius: 3px;
	/* 背景色定为白色吧 */
    background: white;
	/* 文本的大小 */
    font-size: 14px ;
}
 
QLineEdit:hover {
	/* 鼠标悬停时，我们将编辑框的边框设置为蓝色 */
    border: 1px solid blue; 
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>积分（I）：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>400</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLineEdit" name="dLineEdit">
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	/* 边框1px宽，颜色为深紫色 */
    border: 1px solid rgb(41, 57, 85);
	/* 给定3px边框圆角 */
    border-radius: 3px;
	/* 背景色定为白色吧 */
    background: white;
	/* 文本的大小 */
    font-size: 14px ;
}
 
QLineEdit:hover {
	/* 鼠标悬停时，我们将编辑框的边框设置为蓝色 */
    border: 1px solid blue; 
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>微分（D）：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>X: 0.00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="yLabel">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Y: 0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>451</y>
      <width>158</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QPushButton" name="readPidButton">
       <property name="text">
        <string>读取PID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="writePidButton">
       <property name="text">
        <string>写入PID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="resButton">
       <property name="text">
        <string>恢复出厂</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>设置出厂</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openSerialButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>522</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(197, 212, 244);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color: rgb(162, 188, 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>523</y>
      <width>287</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="baudComboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;
	max-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(://image/down_arrow_16.png);
}
</string>
       </property>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="portComboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;
	max-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(://image/down_arrow_16.png);
}
</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="portLabel">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择串口：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="baudLabel">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择波特率：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="loadAction"/>
    <addaction name="screenAction"/>
    <addaction name="aboutAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="newAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建动作</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存数据</string>
   </property>
   <property name="toolTip">
    <string>保存数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="text">
    <string>载入数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="screenAction">
   <property name="text">
    <string>截图</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
